src/connectors/kafka/config.rs

```diff
@@ -0,0 +1,645 @@
+/*
+ * Parseable Server (C) 2022 - 2024 Parseable, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+use anyhow::bail;
+use rdkafka::Offset;
+use rustls::pki_types::{CertificateDer, PrivateKeyDer};
+use rustls_pemfile::{self, Item};
+use serde::{Deserialize, Serialize};
+use std::fmt::Display;
+use std::io::BufReader;
+use std::sync::Arc;
+use tracing::{debug, info};
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct KafkaConfig {
```

Comment:
I wouldn't want to serialize this back into a file, but I would love to extract it directly from the env vars using clap, can we do that? refer: [S3Config](https://github.com/parseablehq/parseable/blob/main/src/storage/s3.rs#L69)

Reply:
Same for all other config types that have to be extracted from env vars